src/corppa/config.py

```diff
@@ -2,16 +2,22 @@
 Load local configuration options
 """
 
-import configparser
 import pathlib
 
+import yaml
+
+try:
+    from yaml import CLoader as Loader
+except ImportError:
```

Comment:
Would it be reasonable to move the "pragma: no cover" to include the except clause too?

```suggestion
except ImportError:  # pragma: no cover
```

Reply:
ah yep, that makes sense - thanks